src/test/java/de/retest/web/ShowcaseIT.java

```diff
@@ -31,6 +31,8 @@ void showcase_html_should_be_checked( final WebDriver driver ) throws Exception
 		final Path showcasePath = Paths.get( "src/test/resources/pages/showcase/retest.html" );
 		driver.get( showcasePath.toUri().toURL().toString() );
 
+		Thread.sleep( 1000 );
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "Thread.sleep()". ([squid:S2925](https://sonarcloud.io/organizations/retest/rules#rule_key=squid%3AS2925))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.retest%3Arecheck-web&pullRequest=84&issues=AWahP8EmrChX7_AlIBKE&open=AWahP8EmrChX7_AlIBKE)

[ISSUE_KEY]: AWahP8EmrChX7_AlIBKE